README.md

```diff
@@ -1,17 +1,19 @@
-# Netflix Conductor SDK
+# Orkes Conductor Client
 
-`conductor-java` repository provides the client SDKs to build Task Workers in Java
+This repository provides a Java client for Orkes Conductor Server. With this client you can manage
+metadata (Workflows and tasks), run workflows, create workers and more. 
 
-## Quick Start
+## Examples
 
-1. [Setup package](#Setup-conductor-python-package)
-2. [Create and run task workers](docs/worker/README.md)
-3. [Create workflows using code](docs/workflow/README.md)
-4. [Create Kafka queue configuration using code](docs/queue/kafka.md)
```

Comment:
This 4th is a good quick-guide but having it as a "Quick Start" step doesn't seem appropriate